packages/backend/src/schedulerMetrics.ts

```diff
@@ -0,0 +1,25 @@
+import opentelemetry, { ValueType } from '@opentelemetry/api';
+import { schedulerClient } from './clients/clients';
+import { SchedulerWorker } from './scheduler/SchedulerWorker';
+import { VERSION } from './version';
+
+const meter = opentelemetry.metrics.getMeter('lightdash-worker', VERSION);
+const queueSizeCounter = meter.createObservableUpDownCounter<{
```

Comment:
Creates a metric `queue.jobs.count` with 4 possible series (from 2 boolean labels). Use UpDownCounter, which is a metric that increments / decrements in whole numbers and can be added (i.e. we can sum all customers queues and we have total of all jobs in queues).